src/doc/gui_manual/Operators/OperatorTypes/Tessellate_operator.rst

```diff
@@ -0,0 +1,56 @@
+.. _Tessellate operator:
+
+Tessellate operator
+~~~~~~~~~~~~~~~~~~~
+
+The Tessellate operator is an operator that tessellates high order elements
+so that they appear curved.
+
+.. _tessellate:
+
+.. figure:: images/tessellate.png
+
+   Tessellate operator example
+
+The Tessellate operator supports the following high order element types.
+
+* QUADRATIC_EDGE
+* CUBIC_LINE
+* LAGRANGE_TRIANGLE
+* QUADRATIC_TRIANGLE
+* BIQUADRATIC_TRIANGLE
+* LAGRANGE_QUADRILATERAL
+* BIQUADRATIC_QUAD
+* QUADRATIC_QUAD
+* LAGRANGE_TETRAHEDRON
+* QUATRADIC_TETRA
+* LAGRANGE_HEXAHEDRON
+* QUADRATIC_HEXAHEDRON
+
+If the Tessellate operator encounters an unsupported element type it will
+remove the element from the mesh.
+
+Changing the tessellation accuracy
+""""""""""""""""""""""""""""""""""
+
+The tessalation accuracy is controlled by the **Chord error**. It is the
+ratio of a chord to the distance from the curve and is independent of the
+scale of the object. The default **Chord error** is 0.035, which will
+typically still exhibit some discritization. A **Chord error** of 0.01
```

Comment:
spelling: discretization
Why use 0.035 as default if 0.01 eliminates discretization artifacts?

Reply:
That is the default for the VTK filter. Thinking about it further, my test cases had extremely curved elements that required using 0.01 to fully capture the curvature. 0.035 is most likely sufficient in most cases. Making that value smaller will increase the number of cells that an element is tessellated into and the number can go up quickly into the hundreds. I will adjust the documentation.